evap/staff/templates/staff_questionnaire_index_list.html

```diff
@@ -26,6 +31,14 @@
                                 {% blocktrans count questionnaire.questions.all.count as count %}{{ count }} question{% plural %}{{ count }} questions{% endblocktrans %},
                                 {% blocktrans count questionnaire.contributions.count as count %}used {{ count }} time{% plural %}used {{ count }} times{% endblocktrans %}
                             </td>
+                            <td>
+                                {% if type != 'contributor' %}
+                                    <div class="btn-switch btn-group icon-buttons">
+                                        <button data-is-locked="0" type="button" onclick="changeLocked($(this));" class="btn btn-sm btn-light{% if questionnaire.is_locked == False %} active{% endif %}" data-toggle="tooltip" data-placement="top" title="{% trans 'Allow editors to change the selection of this questionnaire' %}"><span class="fas fa-fw fa-lock-open" aria-hidden="true"></span></button>
+                                        <button data-is-locked="1" type="button" onclick="changeLocked($(this));" class="btn btn-sm btn-light{% if questionnaire.is_locked == True %} active{% endif %}" data-toggle="tooltip" data-placement="top" title="{% trans 'Prevent editors from changing the selection of this questionnaire' %}"><span class="fas fa-fw fa-lock" aria-hidden="true"></span></button>
```

Comment:
@janno42 I think the icons are clear, but the texts have the common problem that a user can not distinguish between: "This is the current state" vs "When I click/switch this, I will achieve what the text says"

Reply:
we found it's fine here because there are two buttons always shown next to each other.